strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/PypiFileSystem.java

```diff
@@ -0,0 +1,37 @@
+package org.carlspring.strongbox.providers.layout;
+
+import org.carlspring.strongbox.providers.io.LayoutFileSystem;
+import org.carlspring.strongbox.providers.io.StorageFileSystemProvider;
+import org.carlspring.strongbox.storage.repository.Repository;
+
+import javax.inject.Inject;
+import java.nio.file.FileSystem;
+import java.util.Set;
+
+/**
+ * @author carlspring
+ */
+
+/*
```

Comment:
Why is this class commented out?
(I'm actually shocked it compiles...?)

Reply:
When it is not commented out I get this error on line 27:

"The constructor LayoutFileSystem(Repository, FileSystem, StorageFileSystemProvider) is undefined"

Reply:
Interesting. It compiles fine for me and I don't seem to have such issues.

@sbespalov : As an Eclipse user, do you have any thoughts about this one in Eclipse?